### PR TITLE
feat(minnaker): Add recipe for minnaker

### PR DESCRIPTION
**Pull request by alchen99**

### CODE REVIEW

File: recipes/kustomization-minnaker.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+resources:
+  - spinnakerservice.yml                    # (Mandatory). Base spinnaker manifest
+  - infrastructure/minio.yml                # Self hosted minio, a S3 compatible data store
+  - accounts/kubernetes/spin-sa.yml         # Kubernetes service account needed by patch-kube.yml
+  - expose/ingress-traefik.yml              # Use Traefik for Ingress
+
+patchesStrategicMerge:
+  - persistence/patch-minio.yml             # (Mandatory). Persistence to store spinnaker applications and pipelines
+  - expose/patch-ingress.yml                # Automatically expose spinnaker
+  - expose/patch-urls.yml                   # Deck and Gate URLs
```
Comment (away168) [inline recipes/kustomization-minnaker.yml:33]: i think we can comment out the patch-urls.yml 
also i think expose/patch-ingress will handle the urls if it finds them in the ingress definition.

**Comment by away168** [comment]
tangentially, i'm interested in updating github.com/armory/minnaker to use this repo. 
@alchen99 interested in helping?

File: recipes/kustomization-minnaker.yml (hunk 1 above)
Comment (german-muzquiz) [reply recipes/kustomization-minnaker.yml:33]: This is correct. Probably needs to be tested though with traefik.

File: recipes/kustomization-minnaker.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#-----------------------------------------------------------------------------------------------------------------------
+# Quick Start recipe
```
Comment (german-muzquiz) [inline recipes/kustomization-minnaker.yml:2]: Title needs to be updated

File: recipes/kustomization-minnaker.yml (hunk 1 above)
Comment (alchen99) [reply recipes/kustomization-minnaker.yml]: @away168 @german-muzquiz Removed patch-urls and updated ingress-traefik. That combination works.

File: recipes/kustomization-minnaker.yml (hunk 2 above)
Comment (alchen99) [reply recipes/kustomization-minnaker.yml]: @german-muzquiz Title update.

**Comment by alchen99** [comment]
> tangentially, i'm interested in updating github.com/armory/minnaker to use this repo.
> @alchen99 interested in helping?
> making a new branch to make the changes : https://github.com/armory/minnaker/tree/operator-v2

@away168 yes I can help you update the minnaker branch.

**Review by german-muzquiz — APPROVED** [review]

**Pull request merged**